### ISSUE TITLE
How to change pool?

### ISSUE THREAD

**Issue by babasa** [ProgrammerDan/arionum-java#12]
Hi i want to ask is there any ways to change pool, without need to reinstall?

**Comment by ProgrammerDan** [comment]
Yes, simply modify the config.cfg in the same folder on your computer as
the program, the first line is which pool will be used. After the
config.cfg has been changed and saved, restart the miner to start using the
new pool.
